src/main/java/kg/jarkyn/server/Response/ResponseContent.java

```diff
@@ -0,0 +1,20 @@
+package kg.jarkyn.server.Response;
+
+public class ResponseContent {
+    private byte[] byteContent;
+    private String contentType;
+
+    public ResponseContent(byte[] byteContent, String contentType) {
+
+        this.byteContent = byteContent;
```

Comment:
ha ha ha I thought Hungarian is when you indicate type in the name :)

and yeah Response.ResponseContent is an overkill